install.ps1

```diff
@@ -0,0 +1,56 @@
+If (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(`
+    [Security.Principal.WindowsBuiltInRole] 'Administrator'))
+{
+    Write-Warning “You do not have Administrator rights to run this script!`nPlease re-run this script as an Administrator!”
+    Break
+} else {
+
+    Write-Host "Downloading VirtualBox, VirtualBox Extensions, and Vagrant"
+
+
```

Comment:
You are saving `.exe` etc. directly in C drive. I think a better practice would be to use `%APPDATA% or some standard hidden directory for storing these installers temporarily. And once done with the installations, delete them.